README.md

```diff
@@ -108,46 +126,61 @@ int main()
 
 ## Building the library
 
-A compiler that supports **C90 or later** such as *gcc* is required to build the library.
+A compiler that supports **C90 or later** such as _gcc_ is required to build the
+library.
 
-Additionally, the library uses a header file introduced in ISO C99, `stdint.h`. For compilers that do not provide this header file, the [source/include](source/include) directory contains [stdint.readme](source/include/stdint.readme), which can be renamed to `stdint.h` to
-build the backoffAlgorithm library.
+Additionally, the library uses a header file introduced in ISO C99, `stdint.h`.
+For compilers that do not provide this header file, the
+[source/include](source/include) directory contains
+[stdint.readme](source/include/stdint.readme), which can be renamed to
+`stdint.h` to build the backoffAlgorithm library.
+
+For instance, if the example above is copied to a file named `example.c`, _gcc_
+can be used like so:
 
-For instance, if the example above is copied to a file named `example.c`, *gcc* can be used like so:
 ```bash
 gcc -I source/include example.c source/backoff_algorithm.c -o example
 ./example
 ```
 
-*gcc* can also produce an output file to be linked:
+_gcc_ can also produce an output file to be linked:
+
 ```bash
 gcc -I source/include -c source/backoff_algorithm.c
 ```
 
 ## Building unit tests
 
 ### Checkout Unity Submodule
-By default, the submodules in this repository are configured with `update=none` in [.gitmodules](.gitmodules), to avoid increasing clone time and disk space usage of other repositories (like [amazon-freertos](https://github.com/aws/amazon-freertos) that submodules this repository).
 
-To build unit tests, the submodule dependency of Unity is required. Use the following command to clone the submodule:
+By default, the submodules in this repository are configured with `update=none`
+in [.gitmodules](.gitmodules), to avoid increasing clone time and disk space
+usage of other repositories (like
+[amazon-freertos](https://github.com/aws/amazon-freertos) that submodules this
```

Comment:
Isn't this repo deprecated? Should this still be mentioned?

Reply:
Limiting this PR to just changes to the CI-CD actions, spelling, and formatting issues.
This link could be updated to mention FreeRTOS/FreeRTOS instead at this point, but that change is outside the scope of this PR.